src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -222,7 +221,7 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.InstanceId == "" {
-		cfg.InstanceId = hex.EncodeToString(uuid.NodeID())
+		cfg.InstanceId = uuid.New()
```

Comment:
We're now using this as the client ID (cid) @atavism. From the discussion [here](https://groups.google.com/forum/#!topic/google-analytics-measurement-protocol/LN8WfJAPla8) it sounds like the whole cid=555 thing might be the reason we've seen data for a day or so with a lot of these but then somehow have seen that data totally stop.

I changed this to a version 4 UUID because that's what analytics expects, and there's no reason not to use version 4 for our normal InstanceId.